### DISCUSSION TITLE
Impossible to unsubscribed repo alert

### DISCUSSION THREAD

**Discussion by aragon5956** [community/community#141271] [Other Feature Feedback, Questions, & Ideas]
Hello i want to unsubscribed to `php-src` alert and notifications repository and i can't It's impossible 
Regards

**Comment by longnh238** [comment]
Can you do as follows to unsubscribe via GitHub Website:
    - Go to the php-src repository on GitHub.
    - In the upper right corner of the repository page, you should see a "Watch" or "Unwatch" button.
    - Click it and select "Ignoring" or "Unwatch" to stop receiving notifications.

**Reply by aragon5956** [reply]
I dont understand !! Because i did it !!![Screenshot_2024-10-14-17-55-49-55_3aea4af51f236e4932235fdada7d1643.jpg](https://github.com/user-attachments/assets/6141a6a3-6bb0-4877-871f-a6b005f6d866)

**Comment by aragon5956** [comment]
If i click on Custom i've this

![Screenshot_2024-10-14-17-57-02-69_3aea4af51f236e4932235fdada7d1643.jpg](https://github.com/user-attachments/assets/176586a4-e684-40a5-84f8-2e958e09b6f7)
